bpfd/src/command.rs

```diff
@@ -135,6 +135,14 @@ impl std::fmt::Display for Direction {
     }
 }
 
+#[derive(Debug, Clone)]
+pub(crate) struct ListInfo {
+    pub(crate) bpfd_info: Option<ProgramInfo>,
+    pub(crate) info: LinuxProgramInfo,
+}
+
+/// ProgramInfo stores information about bpf programs loaded on a system
+/// which are managed via bpfd.
 #[derive(Debug, Clone)]
 pub(crate) struct ProgramInfo {
     pub(crate) id: Uuid,
```

Comment:
You're going to ask - What UUID should I use here?
If we write one out to disk somewhere - i.e in `/var/run/bpfd`, we can use that as a namespace to generate v5 UUIDs from the ID and Tag of the programs in kernel (which are the uniquely identifying characteristics). You could even include load time in the hash if you wanted. 

```rust
use uuid::{Uuid}; // 1.4.0

fn main() {
    let host_uuid = Uuid::new_v4(); // populated at boot, stored on disk
    println!("Host UUID: {}", host_uuid);
    
    let id = 42u32;
    let tag = u64::MAX;
    
    let data = format!("{}:{:x}", id, tag); // could just use bytes instead of formatting
    println!("Data to be hashed: {}", data);
    let uuid = Uuid::new_v5(&host_uuid, &data.into_bytes()); 
    println!("{}", uuid);
}
```

This ID is stable across bpfd reboots because it would be re-generated using the same data.

Reply:
I never needed to generate a UUID, since we can't really do that deterministically since now users can provide their own UUID.  Therefore, I kept the bpfd UUID and Kernel_ID as  two separate entities.